### PR TITLE
Add support for configuration of resolution for the e-Ink display

### PR DESCRIPTION
**Pull request by lifeofbrian**
I added a width and heigh configuration variable, as the v2 version of the 7.5" screen has a higher resolution

### CODE REVIEW

**Comment by mendhak** [comment]
Thanks for this.  While going through this, I had an idea.  If we just get the user to say whether they have V1 or V2 of the EPaper Display, we could do the height/width and Python import for them.  That way, less work for them to do.  I'm trying this out right now.  I'll see if I can push to this branch.

**Comment by mendhak** [comment]
@lifeofbrian  have a look, what do you think now?

File: display.py [hunk 1]
```diff
@@ -11,9 +11,11 @@
 
 import logging
 
-##If you have a Waveshare v2 screen, try changing this to 
-#from waveshare_epd import epd7in5_V2 as epd7in5
-from waveshare_epd import epd7in5 as epd7in5
+if os.getenv("WAVESHARE_EPD75_VERSION",1):
+    from waveshare_epd import epd7in5 as epd7in5
+else:
+    from waveshare_epd import epd7in5_V2 as epd7in5
```
Comment (mendhak) [inline display.py:17]: Based on the var, the right Python module should be imported

File: run.sh [hunk 2]
```diff
@@ -14,7 +14,16 @@ sudo -E python3 screen-calendar-get.py
...
 # Inkscape can't export to BMP, so let's export to PNG first. 
-inkscape  screen-output-weather.svg --without-gui -e screen-output.png -w640 -h384 --export-dpi=150
+
+if [ $WAVESHARE_EPD75_VERSION = 1 ]; then 
+    WAVESHARE_WIDTH=640
+    WAVESHARE_HEIGHT=384
+else
+    WAVESHARE_WIDTH=800
+    WAVESHARE_HEIGHT=480
+fi
```
Comment (mendhak) [inline run.sh:24]: And here we set the height width.  So the user just has to modify one line in env.sh

Comment (lifeofbrian) [reply run.sh:24]: yeah -- this seems reasonable.

File: display.py (hunk 1 above)
Comment (lifeofbrian) [reply display.py:17]: I may suggest setting the default to "2", as that is the model that is actively being sold on Amazon right now for newbies like me.

Comment (mendhak) [reply display.py:17]: Yes good point. I will change it now.

**Comment by mendhak** [comment]
Merged.  Woger!

**Pull request merged**